tools/accuracy_checker/accuracy_checker/adapters/detection.py

```diff
@@ -852,3 +852,55 @@ def _extract_predictions(self, outputs_list, meta):
                 total_boxes = np.concatenate((total_boxes, boxes), axis=0)
 
         return [total_boxes]
+
+
+class FCOSPersonAdapter(Adapter):
+    """
+    Class for converting output of FCOS model to DetectionPrediction representation
+    """
+    __provider__ = 'fcos_person'
+    prediction_types = (DetectionPrediction, )
+
+    def validate_config(self):
+        super().validate_config(on_extra_argument=ConfigValidator.ERROR_ON_EXTRA_ARGUMENT)
+
+    @classmethod
+    def parameters(cls):
+        parameters = super().parameters()
+        parameters.update({
+            'output_blob': StringField(optional=True, default=None, description="Output blob name."),
+            'scale': NumberField(optional=True, default=1.0, description="Scale factor for bboxes."),
+        })
+
+        return parameters
+
+    def configure(self):
+        self.out_blob_name = self.get_value_from_config('output_blob')
+        self.scale = self.get_value_from_config('scale')
+
+    def process(self, raw, identifiers=None, frame_meta=None):
+        """
+        Args:
+            identifiers: list of input data identifiers
+            raw: output of model
+        Returns:
+            list of DetectionPrediction objects
+        """
+
+        if self.out_blob_name is None:
+            self.out_blob_name = self.output_blob
```

Comment:
I think better to move it to `configure` method. @eaidova, are you agree?

Reply:
Also, could `self.output_blob` has `None` or another inappropriate value?

Reply:
please look my previous comment why it  should be here, not in onfiguration